src/Core/Services/Implementations/SendService.cs

```diff
@@ -280,6 +280,23 @@ private async Task ValidateUserCanSaveAsync(Guid? userId)
                     throw new BadRequestException("Due to an Enterprise Policy, you are only able to delete an existing Send.");
                 }
             }
+
+            if (send.HideEmail.GetValueOrDefault())
+            {
+                foreach (var policy in policies.Where(p => p.Enabled && p.Type == PolicyType.SendOptions && !_currentContext.ManagePolicies(p.OrganizationId)))
+                {
+                    SendOptionsPolicyData data = null;
+                    if (policy.Data != null)
+                    {
+                        data = JsonConvert.DeserializeObject<SendOptionsPolicyData>(policy.Data);
+                    }
+
+                    if (data?.DisableHideEmail ?? false)
+                    {
+                        throw new BadRequestException("Due to an Enterprise Policy, you are not allowed to hide your email address on a Send.");
```

Comment:
Please consider: "Due to an Enterprise Policy, you are not allowed to hide your email address from recipients when creating or editing a Send."

OR whatever I had suggested in the clients themselves, don't remember.